### DISCUSSION TITLE
How can I add an program that is already installed to phoenicis?

### DISCUSSION THREAD

**Discussion by ghost** [PhoenicisOrg/phoenicis#2321] [Q&A]
Some windows programs don't come with a installer,they are pre installed and the .exe opens the program itself.
So how can I add it if it isn't in a prefix?
And how can I add it if it's already in a prefix (for example a steam game mod launcher that is in the steam game prefix)?

**Comment by plata** [comment]
You can add new shortcuts in the library tab.

**Reply by ghost** [reply]
How can I do that there?

**Reply by plata** [reply]
You need to be in advanced mode. Then you will have a button.

**Reply by ghost** [reply]
With "shortcuts",does that mean that I can just add an executable and select the wine version it uses and then select a icon and use it?

**Reply by plata** [reply]
Yes, basically.

**Reply by ghost** [reply]
And if I do a shortcut,does it move the selected executable to the prefix?

**Reply by plata** [reply]
No. It runs the executable inside the prefix. It doesn't move anything.
